app/src/main/java/org/mozilla/rocket/privately/browse/BrowserFragment.kt

```diff
@@ -229,6 +232,19 @@ class BrowserFragment : LocaleAwareFragment(),
     }
 
     class Observer(val fragment: BrowserFragment) : SessionManager.Observer, Session.Observer {
+        override fun updateFailingUrl(url: String?, updateFromError: Boolean) {
```

Comment:
I feel like we can have one default observer here since they all do nothing

Reply:
I tried to use java8 default methods but it needs some extra flags. I think a default observer is not feasible here given that there are interfaces that extends client so that would make the code even harder to read.